css/style.css

```diff
@@ -0,0 +1,655 @@
+  color: #fff;
+  transition: 0.3s;
+}
+
+.main__navMenu ul li a {
+  color: #272a31;
+  font-weight: 500;
+  transition: 0.3s ease;
+}
+
+.socialIcon a:hover {
+  color: #ec5242;
+  transition: 0.3s;
+}
+
+.sub__navMenu ul {
+  display: flex;
+  gap: 30px;
+}
+
+.sub__navMenu ul li a:hover {
+  color: #ec5242;
+}
+
+.main__navbar {
+  height: 600px;
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  background: #fff;
+}
+
+.main__navMenu ul {
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.close button,
+.more button {
+  padding: 20px;
+  width: 100%;
+  background: #fff;
+  border: 2px solid #d3d3d3;
+  transition: 0.3s;
+}
+
+.main__navMenu ul li button {
+  width: 150px;
+  height: 50px;
+  border: 5px solid #ec5242;
+  background: #fff;
+  color: #ec5242;
+  transition: 0.3s ease-in-out;
+}
+
+.more button:hover,
+.close button:hover {
+  background: var(--black);
+  color: var(--grey);
+}
+
+.main__navMenu ul li a:hover {
+  color: #ec5242;
+}
+
+.main__navMenu ul li button:hover {
+  color: #fff;
+  background: #d3d3d3;
+}
+
+/* -----------------------------Navbar Section Ends------------------------------------------------------ */
+
+/* -----------------------------Header Banner Section Start---------------------------------------------- */
+
+.headerBanner {
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  gap: 7px;
+  background-image: url(/Module1--CapstoneProject/img/1025341.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.cross {
+  position: absolute;
+  top: 14px;
+  left: 20px;
+  color: #272a31;
+  font-size: 40px;
+}
+
+.banner {
+  margin-top: 20px;
+}
+
+.subHeader {
+  color: #ec5242;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.mainHeader {
+  font-weight: 800;
+  font-size: 33px;
+  background-image: url(/Module1--CapstoneProject/img/water.jpeg);
+  background-size: contain;
+  background-clip: var(--grey);
+  -webkit-background-clip: text;
+  color: transparent;
+}
+
+.headerPara {
+  margin-top: 20px;
+  padding: 10px;
+  background: #d3d3d39a;
+  border: 3px solid aliceblue;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 25px;
+}
+
+.placeNdate {
+  padding: 5px;
+}
+
+.dateTag {
+  font-size: 25px;
+  font-weight: 800;
+  color: #ec5242;
+}
+
+.datePara {
+  font-weight: 400;
+  font-size: 15px;
+  color: #d3d3d3;
+}
+
+/* -----------------------------Header Banner Section End--------------------------------------------------- */
+
+/* -----------------------------Program Section Starts--------------------------------------------------------- */
+
+.program {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 20px;
+  background: url(/Module1--CapstoneProject/img/pexels-photo-11196384.jpeg);
```

Comment:
- [x] Here once again: 🤓 

![image](https://user-images.githubusercontent.com/77069011/204028170-4ca90373-589c-4cce-8733-8cb3e42a495a.png)

```suggestion
  background: url(/img/pexels-photo-11196384.jpeg);
```